Tempo/Controllers/FeedViewController.swift

```diff
@@ -159,6 +160,25 @@ class FeedViewController: PlayerTableViewController, SongSearchDelegate, PostVie
 		refreshFeedWithDelay(1.0, timeout: 5.0)
 	}
 	
+	func continueAnimatingAfterRefresh() {
+		//animate currently playing song
+		if let playerCellPost = (self.navigationController as! PlayerNavigationController).playerCell.post {
+			let playingSong = playerCellPost.song
+			for row in 0 ..< posts.count {
+				let cell = tableView.cellForRowAtIndexPath(NSIndexPath(forRow: row, inSection: 0)) as! FeedTableViewCell
+				if let cellPost = cell.postView.post {
```

Comment:
should squash the indentation livel

if let cellPost = cell.postView.post where cellPost.song.spotifyID == playingSong.spotifyID {
    //animate the appropriate post view
}